axelor-talent/src/main/resources/i18n/messages_fr.csv

```diff
@@ -93,11 +93,11 @@
 "Hiring stage","Etape de recrutement",,
 "Invalid dates. From date must be before to date.",,,
 "Is template",,,
-"Job Application",,,
+"Job Application","Candidature",,
 "Job Application Filters",,,
 "Job Offer Information","Informations de l'offre d'emploi",,
 "Job Position Filters",,,
-"Job application","Candidature",,
+"Job application","Candidatures",,
```

Comment:
You have to translate "Job application" to "Candidature" and "Job applications" to "Candidatures"